gui/packages/ubuntupro/lib/pages/subscription_status/subscribe_now_page.dart

```diff
@@ -115,6 +115,8 @@ extension PurchaseStatusl10n on PurchaseStatus {
         return lang.purchaseStatusServer;
       case PurchaseStatus.unknown:
         return lang.purchaseStatusUnknown;
+      default:
+        throw UnimplementedError(toString());
```

Comment:
Inacessible by design, only being caught if the enum is changed without this function being updated.